src/routes/strategies/[strategy]/StrategyNav.svelte

```diff
@@ -1,38 +1,160 @@
 <script lang="ts">
-	import { Menu, MenuItem } from '$lib/components';
+	import { Button, Menu, MenuItem } from '$lib/components';
+	import windowInnerWidth from '$lib/window-inner-width';
 
 	export let strategyId: string;
 	export let currentPath: string;
+
+	let detailsEl: HTMLDetailsElement;
+
+	$: tag = $windowInnerWidth <= 1024 ? 'details' : 'aside';
```

Comment:
Per my earlier comment regarding CLS – I may end up changing this later to a CSS-only solution. Let's stick with it as-is for now, though (I don't want this _possible_ issue to hold up the PR).

Reply:
yes, of course, i just treated this as an ad-hoc solution and obviously we need to aim to have CLS equal to 0